specs/agents/tracing-spans.md

```diff
@@ -54,7 +54,36 @@ The documentation should clarify that spans with `unknown` outcomes are ignored
 
 ### Span stack traces
 
-Spans may have an associated stack trace, in order to locate the associated source code that caused the span to occur. If there are many spans being collected this can cause a significant amount of overhead in the application, due to the capture, rendering, and transmission of potentially large stack traces. It is possible to limit the recording of span stack traces to only spans that are slower than a specified duration, using the config variable `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION`.
+Spans may have an associated stack trace, in order to locate the associated
+source code that caused the span to occur. If there are many spans being
+collected this can cause a significant amount of overhead in the application,
+due to the capture, rendering, and transmission of potentially large stack
+traces. It is possible to limit the recording of span stack traces to only
+spans that are slower than a specified duration, using the config variable
+`span_stack_trace_min_duration`. (Previously
+`span_frames_min_duration`.)
+
+#### `span_stack_trace_min_duration` configuration
+
+Sets the minimum duration of a span for which stack frames/traces will be
+captured.
+
+This option is case-insensitive.
```

Comment:
You mean the values are case-insensitive? I.e. I can use `8ms` or `8MS`?  If so, perhaps we could update the Duration description in the "Configuration Value Types" section above to mention case-insensitivity. Currently the Node.js agent is *not* allowing uppercase letters for the units.

Reply:
I'm not aware of other agents that allow for case-insensitive duration suffixes. When we first introduced the suffixes, we intentionally wanted to be quite strict on the parsing rules to ensure each agent has the same behavior. That also includes not allowing whitespaces such as `1 ms`.

Reply:
+1 to being strict here. SI unit prefixes imply case-sensitivity -- though I'll grant users will be unlikely to read "10Ms" as 10 mega-seconds. :)